packages/tpetra/core/test/ImportExport2/ImportExport2_UnitTests.cpp

```diff
@@ -2008,6 +2010,123 @@ TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( Import_Util, UnpackAndCombineWithOwningPIDs,
 }
 
 
+// Unit Test the functionality in Tpetra_Import_Util
+TEUCHOS_UNIT_TEST_TEMPLATE_3_DECL( Import_Util, SortCrsEntries, LO, GO, Scalar)  {
```

Comment:
You may like to include this in a separate .cpp test file.  I like having more, shorter test files.  It makes them easier to read, and may improve build times.